### ISSUE TITLE
doc/core/examples/simple.tac is broken

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3173]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#3173|
|Type|defect|
|Created|2008-04-08 14:32:45Z|

It uses removed twisted.application.compat.

Attachments:

* [simple-tac-3173.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/070/070712c23d5a7197bce22cc097c928860c96187b/5f9df4cd87ed5c45909d8c71b7abaad988aa11c4.patch) (1427 bytes) - added by thijs on 2008-07-13 09:23:47Z - patch against r24264

<details><summary>Searchable metadata</summary>

```
trac-id__3173 3173
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1207665165000000 1207665165000000
changetime__1216001486000000 1216001486000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

File in question: [doc/core/examples/simple.tac simple.tac](doc/core/examples/simple.tac simple.tac)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @therve<br>@thijstriemstra set status to `new`|
|-|-|

The whole purpose of the first server on port 8080 is a little vague to me, but I attached a [[simple-tac-3173.patch](None/simple-tac-3173.patch) patch] that removes the compat import etc.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @thijstriemstra|
|-|-|

Please apply.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24269]) Apply simple-tac-3173.patch: Remove the use of twisted.application.compat in simple.tac example.

Author: thijs
Reviewer: therve
Fixes: [#3173](https://github.com/twisted/twisted/issues/3173)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
